docs/admin/addons.md

```diff
@@ -0,0 +1,37 @@
+---
+---
+
+## Overview
+
+Add-ons extend the functionality of Kubernetes in a pluggable way.
+You can install add-ons on a Kubernetes cluster using `kubectl apply -f`.
+
+This page lists some of the available add-ons.
+
+## Networking
```

Comment:
I've verified that the following networking addons work with kubeadm clusters:

- Calico: https://github.com/projectcalico/calico-containers/tree/master/docs/cni/kubernetes/manifests/kubeadm
- Canal: https://github.com/tigera/canal/tree/master/k8s-install/kubeadm

@lukemarsden how would you like to add these?  Should I just open a PR against your branch?